src/Maestro/Maestro.MergePolicyEvaluation/Maestro.MergePolicyEvaluation.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Please check other projects. This seems to come from a global property

Reply:
Fixed